languages/elixir/reference/exercise-concepts/secret-handshake.md

```diff
@@ -0,0 +1,53 @@
+# Concepts for secret-handshake
+
+## Optimal solution
+```elixir
+defmodule SecretHandshake do
+  use Bitwise
```

Comment:
I assume this is an import of some sorts, in which case the "Imports" Concept should also be added to the list.

Reply:
As above, present in #855